packages/lesswrong/lib/routes.ts

```diff
@@ -1315,7 +1315,7 @@ addRoute(
     path:'/posts/:_id/:slug?',
     componentName: 'PostsSingle',
     titleComponentName: 'PostsPageHeaderTitle',
-    subtitleComponentName: 'PostsPageHeaderTitle',
+    subtitleComponentName: isEAForum ? undefined : 'PostsPageHeaderTitle',
```

Comment:
Does this ever _not_ return null?

Reply:
If I were to be like, "I think this is going to break the `isSubtitle` case" what would you tell me?

Reply:
Hypothetically, if you were to ask that, I'd say that the only place where `subtitleComponentName` is used is in `HeaderSubtitle` where it sets `isSubtitle` to true, so the `if (!isSubtitle)` branch is never used.

So, no, it always returns null, but it's complicated enough that I don't just want to rip it out at midnight on a friday night without thinking about it 😅

Reply:
Lol. Maybe merge and mark as a todo in asana?